src/components/ButtonPanel.js

```diff
@@ -1,74 +1,103 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
+const styles = {
+  display: 'flex',
+  flexDirection: 'column',
+};
+
+const rowStyles = {
+  height: 100,
+  width: '100%',
+  display: 'flex',
+};
+
+const btnStyles = {
+  width: '25%',
+  backgroundColor: '',
+};
+
+let btnWide;
 
 // eslint-disable-next-line  react/prefer-stateless-function
 class ButtonPanel extends React.Component {
   render() {
+    const { color } = this.props;
+    const { wide } = this.props;
```

Comment:
### [Optional]
I think we could have this in one line instead of repeating ourselves. let's practice [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) code.

Example: `const { color, wide } = this.props;`

Reply:
didn't know you can do that, will fix that, thanks!